vue/components/forms/personalization-form/reference.vue

```diff
@@ -156,6 +157,9 @@ export const Reference = {
         configMeta() {
             return this.$store.state.config.meta || {};
         },
+        openExternally() {

```

Comment:
Doesn't this come from props? (like in the personalization plugins for retail-vendors)

Reply:
It's coming from props but only for certain plugins [such as dior](https://github.com/ripe-tech/ripe-retail-vendors/blob/2520d1f426fe7b3c60d47b312f72a3ab4c6cb419/dior/personalization.vue#L3). For every other case, this prop is not being defined which means it will be disabled.

This is to add "open-externally" as a feature in the reference plugin.

Reply:
Yes, the idea is that the concept of features sticks to white and instead components receive it by props.

Reply:
https://github.com/ripe-tech/ripe-commons-pluginus/blob/master/vue/components/organisms/personalization/personalization.vue#L182

Reply:
Currently the param `&features=open-externally` isn't having the desired effect for non-retail plugins but indeed white is already doing the `hasFeature` and passing it as a prop so the fix is just a missing prop here 😄 .